src/data/events.ts

```diff
@@ -81,7 +81,25 @@ export const EVENTS: Event[] = [
             href: new URL('https://csclub-adl.square.site/product/cs-club-pub-crawl-2024-shirt/54'),
```

Comment:
Remove the url part for preordering pub crawl shirts since they have sold out. Also, update the details to "Join us for a night of hopping from one watering hole to the next, spreading good vibes and great drinks all around."